CMake/FindMOBJECT.cmake

```diff
@@ -23,7 +23,7 @@ find_path(ABT_INCLUDE_DIR abt.h
   HINTS ${PC_MOBJECT_INCLUDEDIR} ${PC_MOBJECT_INCLUDE_DIRS}
   PATHS /usr/local/include /usr/include)
 
-find_library(MOBJECT_LIBRARY NAMES mobject-client
+find_library(MOBJECT_LIBRARY NAMES mobject-bedrock
```

Comment:
I think this still needs to be mobject-client rather than mobject-bedrock. mobject-bedrock doesn't really have any of the mobject symbols in it that the VOL would be looking for.